pkg/kudoctl/packages/install/operator.go

```diff
@@ -0,0 +1,63 @@
+package install
+
+import (
+	"fmt"
+
+	"github.com/thoas/go-funk"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+)
+
+func installOperatorAndOperatorVersion(client *kudo.Client, resources packages.Resources) error {
+	if !client.OperatorExistsInCluster(resources.Operator.Name, resources.Operator.Namespace) {
+		if _, err := client.InstallOperatorObjToCluster(resources.Operator, resources.Operator.Namespace); err != nil {
+			return fmt.Errorf(
+				"failed to install %s-operator.yaml in namespace %s: %v",
+				resources.Operator.Name,
+				resources.Operator.Namespace,
+				err)
+		}
+		clog.Printf(
+			"operator.%s/%s created in namespace %s",
```

Comment:
By default k8s and KUDO print the objects as `namespace/name` - let's not break this pattern please as it makes grepping logs easier and more consistent.

Reply:
Well, I counted at least 3 different approaches in KUDO while working on this refactor, `$namespace/$name`, `$resource.$apiversion/$name` and `$name-$resource.yaml`. Not sure if this was intended or just due to a lack of consistency. I'll change all occurences to `$namespace/$name`.

Reply:
Definitely lack of consistency: I tried to use `<namespace>/<name>` at least everywhere in the KUDO manager.